styleguide/_static/idioms/sentinel.py

```diff
@@ -0,0 +1,12 @@
+# Good
+blocks = []
+for block in iter(partial(f.read, 32), ""):
+    blocks.append(block)
+
```

Comment:
hm, to be honest, the bad version is probably easier to understand

Reply:
Noted: I will add a better example +  also will add an explanation.